src/ch00/cairo_cheatsheet/felt.md

```diff
@@ -0,0 +1,12 @@
+# Felt252
+
+Cairo supports Felt as an data type for holding short strings, these short strings have a maximum length of 31 characters. For example:
```

Comment:
This is not exactly correct. Felt252 is the fundamental type and all other types in the core library are derived from it.
You can say that felt252 are used to store short-string representation of up-to 31 characters (because 252 bits of ASCII)